### PR TITLE
DUPLO 9203 Enhance Monitoring

### PR DESCRIPTION
**Pull request by Duplo-nikv**
## Overview

Added support for `enhanced_monitoring` attribute for `duplocloud_rds_instance` resource 
This PR does the following:

- Enhanced monitoring support is added for update context
- Document updated

## Testing performed

- [ ] Using unit tests
- [ ] Manually, on my local system
- [ ✓] Manually, on a remote test system

## Describe any breaking changes

- ...

### CODE REVIEW

**Comment by zafarabbas** [comment]
Task linked: [DUPLO-9203 TF: enhanced monitoring](https://app.clickup.com/t/8655600/DUPLO-9203)

**Review by sp1676 — APPROVED** [review]

**Review by komalkadam-dc — APPROVED** [review]

File: duplocloud/resource_duplo_rds_instance.go [hunk 1]
```diff
@@ -275,6 +275,13 @@ func rdsInstanceSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Computed:    true,
 		},
+		"enhanced_monitoring": {
```
Comment (tahir-duplo) [inline duplocloud/resource_duplo_rds_instance.go:278]: Can you please set `enhanced_monitoring` in read context.

Comment (Duplo-nikv) [reply duplocloud/resource_duplo_rds_instance.go:278]: done

**Review by tahir-duplo — APPROVED** [review]
LGTM

**Pull request merged**